tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -17,6 +17,9 @@ readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 source "${script_dir}/../../scripts/lib.sh"
 
+readonly packaging_root_dir="${repo_root_dir}/tools/packaging"
+patches_path=""
+readonly default_patches_dir="${packaging_root_dir}/kernel/patches"
```

Comment:
And why do we need this one?

Reply:
We are calling get_config_version to get the kernel config version. And without these variables, we get unbound variable error.

Reply:
Please, drop those lines as those are not used yet.

Reply:
Please, drop these lines here as they are not used.